[pyos][bug] pycodestyle detects block params with `E265`

part of: https://github.com/pyOpenSci/software-submission/issues/257#issuecomment-4523576886

pycodestyle is flagging this:

```
#| title: "Mortgage Originations per Capita"  # noqa: E265,E501
#| height: 50%  # noqa: E265,E501
```

like 

```
examples/mortgage-dashboard/index.qmd:77:1: E265 block comment should start with '# '
examples/mortgage-dashboard/index.qmd:78:1: E265 block comment should start with '# '
```

apparently it doesn't read the noqa markers? 

tagging as a bug because this would make it impossible to use that rule with pycodestyle

--- a/src/lintquarto/converter.py
+++ b/src/lintquarto/converter.py
@@ -246,7 +246,7 @@ class QmdToPyConverter:
 
         # If line is a quarto chunk option...
         if stripped.startswith("#| "):
-            self._handle_chunk_option(line, stripped)
+            self._handle_chunk_option(stripped)
         # If line is a comment...
         elif stripped.startswith("#"):
             self._handle_comment_in_options(line)
@@ -276,25 +276,16 @@ class QmdToPyConverter:
         line = self._handle_annotations(line)
         self.py_lines.append(line)
 
-    def _handle_chunk_option(self, line: str, stripped: str) -> None:
+    def _handle_chunk_option(self, stripped: str) -> None:
         """
         Handle a Quarto chunk option line (starting with `#|`).
 
         Parameters
         ----------
-        line : str
-            The original line to process.
         stripped : str
             The line with leading whitespace removed.
 
         """
-        # If we've already identified this chunk as a valuebox chunk,
-        # all subsequent chunk options also become placeholders.
-        if self.current_chunk_is_valuebox:
-            if self.preserve_line_count:
-                self._append_placeholder()
-            return
-
         # Parse and store eval option if present in this line
         self.parse_chunk_eval_option(stripped)
 
@@ -304,19 +295,9 @@ class QmdToPyConverter:
         # Detect Quarto valuebox chunks
         if re.match(r"^content\s*:\s*valuebox\s*$", option_text):
             self.current_chunk_is_valuebox = True
-            if self.preserve_line_count:
-                self._append_placeholder()
-            return
 
-        # Don't append if not preserving line count
-        if not self.preserve_line_count:
-            return
-
-        # Suppress E265 (as will warn for "#|" comment spacing)
-        if self.uses_noqa:
-            line = self._add_noqa(line, ["E265"])
-
-        self.py_lines.append(line)
+        if self.preserve_line_count:
+            self._append_placeholder()
 
     def _handle_comment_in_options(self, line: str) -> None:
         """

--- a/tests/test_converter.py
+++ b/tests/test_converter.py
@@ -143,8 +143,8 @@ PYTHON_CHUNKS = [
         "expected": [
             "# %% [python]",
             " ",
-            "#| echo: false  # noqa: E265,E501",
-            "#| output: asis  # noqa: E265,E501",
+            "# -",
+            "# -",
             "1+1  # noqa: E305,E501",
         ],
     },
@@ -157,16 +157,15 @@ PYTHON_CHUNKS = [
         ],
         "expected": [
             "# %% [python]",
-            "    #| echo: false  # noqa: E265,E501",
+            "# -",
             "    x = 1  # noqa: E305,E501",
         ],
     },
     {
-        "id": "malformed chunk options",
+        "id": "malformed chunk options are unhandled",
         "lines": [
             "```{python}",
             "#|echo: true",
-            " #|   echo: false",
             "# | echo: valid",
             "x = 1",
             "```",
@@ -174,7 +173,6 @@ PYTHON_CHUNKS = [
         "expected": [
             "# %% [python]",
             "#|echo: true",
-            " #|   echo: false  # noqa: E265,E501",
             "# | echo: valid",
             "x = 1  # noqa: E305,E501",
             "# -",
@@ -250,14 +248,14 @@ PYTHON_CHUNKS = [
         ],
     },
     {
-        "id": "chunk options and '#<<' that should not be removed",
+        "id": "chunk options and '#<<'",
         "lines": [
             "```{python}",
             "#| echo: false #<<",
         ],
         "expected": [
             "# %% [python]",
-            "#| echo: false #<<  # noqa: E265,E501",
+            "# -",
         ],
     },
     {
@@ -276,14 +274,14 @@ PYTHON_CHUNKS = [
         ],
     },
     {
-        "id": "chunk options and '# <n>' that should not be removed",
+        "id": "chunk options and '# <n>'",
         "lines": [
             "```{python}",
             "#| echo: false # <1>",
         ],
         "expected": [
             "# %% [python]",
-            "#| echo: false # <1>  # noqa: E265,E501",
+            "# -",
         ],
     },
     {

